### PR TITLE
Fix and Update code

### PR DESCRIPTION
**Pull request by duylong13**

### CODE REVIEW

File: Practice2/README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 *Step two*: How to run the project
 * Turn on Visual Studio Code. Open the folder you just cloned to your computer.
 * Select Terminal on the toolbar or press Ctrl + Shift + `
+* Run command "cd Internship"
```
Comment (vuongnguyen-agilityio) [inline Practice2/README.md:25]: Should add one more step: git checkout practice-two

File: Practice2/src/partials/bot-section.html [hunk 2]
```diff
@@ -15,32 +15,32 @@ <h5 class="bot-main-heading">Hofmann</h5>
   </div>
   <div class="bot-container-secondary">
     <ul class="bot-text-list-sub bot-one">
-      <li class="text-list">About Us</li>
-      <li class="text-list">Explore</li>
-      <li class="text-list">Contact</li>
+      <a href="javascript:void(0)" class="link-success"><li class="text-list">About Us</li></a>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/bot-section.html]: a tag should not cover li. li will cover a tag.

File: Practice2/src/partials/portfolio-section.html [hunk 3]
```diff
@@ -86,11 +86,11 @@ <h5 class="item-six-main-heading">Wild
   <div>
     <ul class="list">
       <i class="arrow left"></i>
-      <li class="number-list number-list-one">1</li>
-      <li class="number-list number-list-two">2</li>
-      <li class="number-list number-list-three">3</li>
-      <li class="number-list number-list-four">4</li>
-      <li class="number-list number-list-five">5</li>
+      <a><li class="number-list number-list-one">1</li></a>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/portfolio-section.html]: The same. a tag cannot cover li tag

**Review by vuongnguyen-agilityio — CHANGES_REQUESTED** [review]

File: Practice2/src/partials/portfolio-section.html (hunk 3 above)
Comment (duylong13) [reply Practice2/src/partials/portfolio-section.html]: Fixed

File: Practice2/src/partials/bot-section.html (hunk 2 above)
Comment (duylong13) [reply Practice2/src/partials/bot-section.html]: Fixed

File: Practice2/README.md (hunk 1 above)
Comment (duylong13) [reply Practice2/README.md:25]: Fixed

**Pull request merged**